secure_message/api_mocks/internal_user_service_mock.py

```diff
@@ -53,8 +53,8 @@ def get_user_details(self, uuid):
         """gets the user details from the internal user service"""
         found = None
         if uuid:
-            if uuid in [NON_SPECIFIC_INTERNAL_USER, BRES_USER]:
-                return InternalUserService.get_non_specific_user_details(uuid)
+            if uuid == NON_SPECIFIC_INTERNAL_USER:
+                return InternalUserService.get_default_user_details(NON_SPECIFIC_INTERNAL_USER)
```

Comment:
Why has this changed from uuid to NON_SPECIFIC_INTERNAL_USER?

Reply:
Is this comment still valid?  Looking at the code, I think this is just removing a step from the process that doesn't need to be done anymore